src/scss/styles.scss

```diff
@@ -0,0 +1,50 @@
+// Bootstrap styles overrides
+@import "variables";
+
+// Bootstrap styles
+@import "../../node_modules/bootstrap/scss/bootstrap.scss";
+
+// Fonts
+@import url("https://use.typekit.net/gpj3tvv.css");
+
+.row {
+  margin: 0;
+}
+
+h1 {
+  font-family: maple-web, serif;
+}
+
+label {
+  display: none;
+}
+
+// Navbar
+i {
+  font-size: 1.5rem;
+  color: #000;
+}
+
+// Customized from Bootstrap formcontrol
+.form-control {
+  &:focus {
+    border: 2px solid $secondary;
+  }
+}
+
+// Fixed bottom post to wall
+.fixed-bottom {
+  left: 50%;
+  transform: translateX(-50%);
```

Comment:
This is not a great approach. Why not use a full width div and `d-flex justify-content-center`?

You could even add `margin: auto` to achieve the same thing.